README.md

```diff
@@ -47,15 +49,11 @@ For technical questions regarding the SDK, please visit the FTC Technology forum
 # Release Information
 **************************************************************************************
 
-Version 3.2 (built on 17.08.02)
-
-This version of the software introduces the "OnBot Java" Development Tool.  Similar to the FTC Blocks Development Tool, the FTC OnBot Java Development Tool allows a user to create, edit and build op modes dynamically using only a Javascript-enabled web browser.
-
-The OnBot Java Development Tool is an integrated development environment (IDE) that is served up by the Robot Controller.  Op modes are created and edited using a Javascript-enabled browser (Google Chromse is recommended).  Op modes are saved on the Robot Controller Android device directly.  
-
-The OnBot Java Development Tool provides a Java programming environment that does NOT need Android Studio.
+### Version 3.2 (built on 17.08.02)
 
+This version of the software introduces the "OnBot Java" Development Tool.  Similar to the FTC Blocks Development Tool, the FTC OnBot Java Development Tool allows a user to create, edit and build op modes dynamically using only a web browser, and does NOT need Android Studio.
```

Comment:
You might want to keep the information about Javascript being required there, on the off chance someone tries to use it using something ridiculous (or they just have Javascript disabled with a whitelist).

Reply:
OK, fixed.